src/central-server/admin-service/core/src/main/java/org/niis/xroad/cs/admin/core/entity/RequestEntity.java

```diff
@@ -66,6 +66,16 @@ public abstract class RequestEntity extends AuditableEntity {
     @Getter
     private int id;
 
+    @Column(name = "server_owner_name")
+    @Getter
+    @Setter
+    private String serverOwnerName;
```

Comment:
How are these getting populated? Legacy implementation did duplicate data in database, we should avoid it whenever it is possible. For management requests we even have a view which is used in lists, it does not use this column.